tests/garage/torch/policies/test_gaussian_mlp_policy.py

```diff
@@ -0,0 +1,100 @@
+import pickle
+from unittest import mock
+
+import numpy as np
+import pytest
+import torch
+
+from garage.torch.policies import GaussianMLPPolicy
+
+
+@mock.patch('garage.torch.policies.gaussian_mlp_policy.GaussianMLPModule')
+def test_policy_get_actions(mock_model):
```

Comment:
please parameterize all your tests cases. you can use https://github.com/rlworkgroup/garage/blob/master/tests/garage/tf/policies/test_gaussian_mlp_policy_with_model.pyy as a guide.

Reply:
fixed